src/eval-frame/components/reps/rep-utils/get-data-table-summary.js

```diff
@@ -0,0 +1,44 @@
+import { range } from "lodash";
+import { getValueSummary } from "./get-value-summary";
+
+import { getObjAtPath } from "./get-child-summaries";
+
+export function getDataTableSummary(rootObjName, path, pageSize, page) {
+  const tabularObj = getObjAtPath(window[rootObjName], path);
+
+  // // FIXME: we may wish to consider making the tables sortable
```

Comment:
oh whoops I missed this note when I made my other comment. Seems like this would be useful to make sure we have down the line.